TESTING_WITH_IAF-TEST.md

```diff
@@ -53,3 +53,6 @@ Press [ Start ], sit back, relax, do some stretches, and let's hope for the best
 ### Troubleshooting
 
 * Some parts of the iaf-test module rely on proprietary modules. To tell Maven that it should download these modules, go to Window > Preferences > Maven > User Settings. If you already have a _settings.xml_ file, press the "Open file" link. Otherwise, browse to _C:/Users/(your name)/.m2/_ and create a _settings.xml_ file. Edit the file by adding your own repository or the [frankframework nexus repository](https://nexus.frankframework.org/content/groups/private/) as [mirror](https://maven.apache.org/guides/mini/guide-mirror-settings.html).
+* When your IP-address is dynamically generated, you may have problems connecting to your database that runs in a docker image. The Larva tests reference the database host by a DNS name, `host.docker.internal`. Docker may not automatically update the IP address to which this name refers when your computer is assigned a new IP address. To see whether you have this issue, do `ping host.docker.internal` in a command prompt. If you cannot reach this address, refresh your docker network. You can do that using docker desktop. Press the cog wheel to the top. In the left-hand menu, choose Resources | Network. Update the docker subnet.
+* If you are using an Oracle database, mind the version of the driver you download. For each combination of a JDK version and an Oracle database version, the driver is different. The filename on the Oracle download page depends only on the JDK version though. You can find the Oracle version we use in `ibis-adapterframework-docker/pom.xml`.
+* If initially the database is not available when Tomcat boots, the Frank!Framework's retry mechanism might fail. You may have to restart Tomcat in Eclipse.
```

Comment:
If this is really the case, then this must be fixed. Otherwise, I can imagine this is overly detailed

Reply:
I observed this while testing, but I wonder whether it can be easily reproduced. I will try once more to test this. If I can reproduce it, I will make an issue.

Reply:
I have created issue https://github.com/ibissource/iaf/issues/3062 about this. Now I will remove the text about rebooting.

Reply:
The text about rebooting Tomcat is gone.